tests/framework/clients/corral/config.go

```diff
@@ -23,6 +23,7 @@ type CorralPackages struct {
 	Cleanup             bool              `json:"cleanup" yaml:"cleanup" default:"true"`
 	Debug               bool              `json:"debug" yaml:"debug" default:"false"`
 	CustomRepo          string            `json:"customRepo" yaml:"customRepo"`
+	SetCorralSSHKeys    bool              `json:"setCorralSSHKeys" yaml:"setCorralSSHKeys" default:"false"`
```

Comment:
should we set some rule for bool names in configs? From readability standpoint, this may get confusing in the future if there's no standardization.

Reply:
That's a good point, in this case what would you suggest? What if we did Set for all of them? SetCleanup, SetDebug, SetCorralSSHKeys?

Reply:
GoLang seems to prefer `is` and `has` it's even in the core modules.

Reply:
I think in this let's go with `has` then.

Reply:
sgtm

Reply:
done